spec/predicates/vuln.md

```diff
@@ -0,0 +1,136 @@
+# Predicate type: Vulnerabilities
+
+Type URI: https://in-toto.io/attestation/vulns/attribute-report
+
+Version: 0.1
+
+## Purpose
+
+The definition of a vulnerability attestation type has been discussed in the past in [in-toto attestation](https://github.com/in-toto/attestation/issues/58) and [issue](https://github.com/sigstore/cosign/issues/442). However we need to identify two different purposes from these initial conversations:
+
+* The definition of a common format to represent the results of a vulnerability report.
+
+* The definition of a certain set of metadata fields that could help to consume these vulnerability attestations from the different scanning tools.
+
+Obviously the first goal is quite challenging and requires a bigger community to agree upon a specific format. As a consequence, the following attestation type focuses on the definition of that common metadata which could enable the beginning of an exportable and manageable vulnerability attestation.
+
+This document describes a vulnerability attestation type to represent vulnerability reports from the scanners in an "exportable" manner and independently of the format chosen to output the results.
+
+## Prerequisites
+
+The in-toto [attestation] framework and a [Vulnerability scanner tool].
+
+## Use Cases
+
+When sharing the results of a vulnerability scan using an attestation, there is certain metadata that is crucial to trust and reuse this information.
+Information about the scanner used during the scanning is relevant to trust these resuls. The state of the vulnerability database used to search for vulnerabilities defines the accuracy of the results. Other metadata information such as the timestamp when the scan finished could define the reusability of these results.
+
+## Model
+
+This is a predicate type that fits within the larger [Attestation] framework.
```

Comment:
I think this section could use more detail about how you expect the attestation to be used.